test/cob_app/upload_test.clj

```diff
@@ -0,0 +1,65 @@
+(ns cob-app.upload-test
+  (:require [speclj.core :refer :all]
+            [cob-app.upload :refer :all]
+            [cob-app.core :refer [initialize handle]]
+            [webserver.mock-socket]))
```

Comment:
I prefer restricting my `:refer :all` use to *only* `speclj.core`, and using something like `[cob-app.upload :as upload]` to alias the namespace. This makes it a lot easier to see, within the namespace, which vars are coming from outside this ns, vs. the ones that are either defined in this ns, or are important/well-known enough to feel like part of the core language (that's my reasoning for the `speclj.core` `:refer` I usually do, anyway).

Would be good to add `[clojure.java.io :as io]` here too.